connector/exception/client_exception.py

```
@@ -0,0 +1,19 @@
+from bs4 import BeautifulSoup
+
+from connector.exception.error_type import ErrorType
+
+
+class ClientException(Exception):
+    def __init__(
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)